### DISCUSSION TITLE
Backend stuff

### DISCUSSION THREAD

**Discussion by hyunjiLeeTech** [1B2K/HealthGuard#55] [Ideas]
March 23, 2021
Decided to use AsyncStorage with react-native-sensitive-info
Database: MogoDB

- react-native-sensitive-info
-  Cloud firebase
-  redux-persist
-  react-native-sqlite-storage

Requirement
- can use on offline
- localStorage on device (not like a BookIt)

**Comment by ElisaNgLi** [comment]
Blog information about AsyncStorage 

https://reactnative.dev/docs/asyncstorage
https://www.reactnativeschool.com/migrating-data-in-asyncstorage
https://heartbeat.fritz.ai/building-offline-react-native-apps-with-asyncstorage-dcb4b0657f93
https://blog.jscrambler.com/how-to-use-react-native-asyncstorage/#:~:text=AsyncStorage%20is%20a%20simple%2C%20asynchronous,a%20key%2Dvalue%20storage%20system.

**Comment by hyunjiLeeTech** [comment]
Information about MongoDB set up in React Native
https://docs.mongodb.com/realm/tutorial/react-native/

**Reply by ElisaNgLi** [reply]
https://docs.mongodb.com/realm/react-native/
